src/lib/token-utils.js

```diff
@@ -40,7 +40,10 @@ const KNOWN_TOKENS_FALLBACK = new Map([
   ],
 ])
 
-const LOCAL_TOKEN_ICONS = new Map([['HNY', honeyIconSvg]])
+const LOCAL_TOKEN_ICONS = new Map([
+  ['HNY', honeyIconSvg],
+  ['CVTN', honeyIconSvg], // TODO: Remove
```

Comment:
what is this for?

Reply:
Ah this is so we can see the token icon (which in this case is honey). Since we are mapping the icon from the token symbol, we wouldn't see it  when using  the Rinkeby env if we don't add the Rinkeby token.

Reply:
ahhh cool